pkg/certificates/certificates_test.go

```diff
@@ -53,7 +78,8 @@ func TestAuthorizationDB_Create(t *testing.T) {
 		t.Fatal(err)
 	}
 	defer func() {
-		_ = authDB.Close()
+		err := authDB.Close()
```

Comment:
You can use `defer ctx.Check(authDB.Close)`

Reply:
Nice